### ISSUE TITLE
7 point survey results only show 5 point scale

### ISSUE THREAD

**Issue by joesaunderson** [PostHog/posthog#25563]
## Bug description

Create a survey with a 7 point number scale, view results.

Results are shown on 5 point scale.

https://eu.posthog.com/project/3310/surveys/0192570d-1dc6-0000-394b-e4b3ddf6452a

## How to reproduce

1. Create a survey with a 7 point number scale, view results.
2. Results are shown on 5 point scale.

## Additional context

It's a CES survey template?

### Debug info

```shell
- [ ] PostHog Cloud, Debug information: [please copy/paste from https://us.posthog.com/settings/project-details#variables or https://eu.posthog.com/settings/project-details#variables]
- [ ] PostHog Hobby self-hosted with `docker compose`, version/commit: [please provide]
- [ ] PostHog self-hosted with Kubernetes (deprecated, see [`Sunsetting Kubernetes support`](https://posthog.com/blog/sunsetting-helm-support-posthog)), version/commit: [please provide]
```

**Comment by Aashw1n** [comment]
Expanded on this problem here, and added a fix : #25702

**Comment by annikaschmid** [comment]
This was fixed here: https://github.com/PostHog/posthog/issues/25702
